src/plone/restapi/tests/http-examples/workingcopy_baseline_get.resp

```diff
@@ -43,14 +43,14 @@ Content-Type: application/json
     "language": "",
     "layout": "document_view",
     "lock": {
-        "created": "1995-07-31T17:30:00Z",
+        "created": "1995-07-31T15:30:00Z",
         "creator": "admin",
         "creator_name": "admin",
         "creator_url": "http://localhost:55001/plone/author/admin",
         "locked": true,
         "name": "iterate.lock",
         "stealable": false,
-        "time": 807211800.0,
+        "time": 807204600.0,
```

Comment:
This might be dependent on the timezone, so if this give test failures, it should be reverted.